### DISCUSSION TITLE
Création d’un profil « référent de formation »

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#11] [6- Administration]
Création d’un profil « référent de formation » permettant à un enseignant de créer/modifier ses créneaux. Ce rôle de « référent de formation » pourrait être donné par le référent lycée.

**Comment by efinkbeiner5** [comment]
Du coup, ce serait que pour les lycées ou aussi pour les établissements avec un référent formation au sein d'une structure ?

**Reply by MagaliMoserNoriaIRL** [reply]
Je pense que cela pourrait être intéressant de l'avoir pour les deux, lycée et structure, afin de donner un peu d'autonomie dans l'organisation interne de la gestion des immersions.

**Reply by SandrineRobert** [reply]
Oui, cette fonctionnalité serait très intéressante côté université de Bordeaux. Notamment pour les secrétaires pédagogiques qui sont les seules à m'aider à repérer les erreurs dans les créneaux de cours : le rôle de référent formation permettrait de mieux cibler leur périmètre que celui de référent structure.

**Comment by efinkbeiner5** [comment]
- Création d'un nouveau groupe d'utilisateurs 
- Possibilité d'affecter les référents de formations par les référents structure via l'application sans passer par l'admin
- Création automatique du compte de l'utilisateur
- Filtrage des listes de cours, créneaux, évènements hors offre selon les droits de l'utilisateur
Chiffrage : Fort

**Reply by efinkbeiner5** [reply]
le référent de formation en lycée peut gérer les créneaux mais n'a pas les droits du référent Lycée

**Comment by efinkbeiner5** [comment]
Non réalisé en 2025
